packages/vuetify/src/composables/validation.ts

```diff
@@ -81,7 +83,11 @@ export function useValidation (
       ? wrapInArray(props.errorMessages).slice(0, Math.max(0, +props.maxErrors))
       : internalErrorMessages.value
   })
+  const isEmpty = computed(() => {
```

Comment:
no need for this computed I think. should be fine to just use `if (model.value)`

Reply:
I did it like that because originally I was also checking for an empty array, but I removed it I don't remember why. I could add it again.

Reply:
For example,
```suggestion
  const isEmpty = computed(() => !props.modelValue || (Array.isArray(props.modelValue) && !props.modelValue.length))
```

Reply:
And I'm not sure doing if(model.value) is good, because we would count 0 as an empty value which I don't think it should be.

Reply:
I have updated isEmpty

Reply:
the model value is always a string though, so 0 shouldn't be an issue

Reply:
Yeah there isn't really a consistent definition for "empty". Text fields would be `''` or `null`, but both are valid select and radio values. `[]` could be empty for multi select but not single select or radio. Really the only empty value everywhere is `undefined` 

[Playground](https://play.vuetifyjs.com/#eNqlkj1v2zAQhv8KwUUJYJIOPNVQjHTt0A4dLQ+MdE7Y8gskpaIQ9N9zlBTHsYNAQCaJx+fe9z647+l373nXAt3SMoHxWibYVZaQsmPS+/F3PNTOJqkshDk0BoNslGNPwbWedMy4BvR9RSNoqBM0FT2xbzTZdlK3gJhttUYkf0ox336acJQ6AmaM32UpRYF8USyD18iul6H7A7L7wzV8ikxTwfjpxgfY9T358fvXTx5TUPZJHf/fvA7rlgxDKTIzjxyF3s88R6aVlOJsVXiMdVA+kQhpdCREGe9CIj0JcCQDOQZnSIFbLqZyUDdmfHIm9xm7uc26k9KOrugkwYz0/E90Ft9Hn1Or+SJWdEvGSI6hdD5X9DklH7dC1I3FNHwOqgvcQhLWG/GAmAitTcoAa5x52PANXzOp/bPk30SjYjq/5hANewzuX4SAYhVdndklnN0Cy5lEpzt+t5ks5hjT8jFmjyttgYYdBBbANhAgLG3tIu2j9i6QqxZzFUNlB1yAVvZvvJh9Hce571+L/Vrvo1qWOmTHYUXfFYwlTJn08ALHJVx8)

Reply:
What is your proposal then? Can we make it component-specific? Each component that uses validation could have a list of 'empty values'.